### ISSUE TITLE
[Support Admin] Verify/Edit organizations required fields can be submitted empty

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#6399]
## Area or Component

Support Admin > Verify/Edit organization  > Verify/Edit modal

https://github.com/CDCgov/prime-simplereport/assets/10108172/33a3ef9e-544a-468c-a4be-c2c109abf266

## Quick Summary

Admin Name, email, and phone number show that they are required but can be submitted with empty data

## Accessibility Violation

https://docs.deque.com/issue-help/1.0.0/en/optional-marked-required

## Action Requested

If possible use react-hook-form to refactor this form and enable validation.

## Source Code (optional)

[which file or line is the issue attached to?]: #

**Comment by mehansen** [comment]
previously reported as a bug in #5867 

copying over the content from the ticket @emyl3 created and closing that one so we can merge to one

> ### Current Behavior
> 
> When editing a pending organization, a support admin can delete values in required fields (e.g. "Administrator first name", "Administrator last name", "Administrator email", "Administrator phone") and submit the form.
> 
> ### Steps To Reproduce
> 
> 1. Go to `/admin/pending-organizations`
> 2. Click "Edit/Verify"
> 3. When the form modal opens, clear all the required fields except "Organization name"
> 4. The form saves with empty required fields
> 
> ### Expected Behavior
> 
> 1. Go to `/admin/pending-organizations`
> 2. Click "Edit/Verify"
> 3. When the form modal opens, clear all the required fields except "Organization name"
> 4. Error messages appear for all the blank required fields
> 
> ### Additional context
> 
> Discovered while testing: https://github.com/CDCgov/prime-simplereport/pull/5838#pullrequestreview-1440627001
